src/test/java/org/folio/linked/data/service/ResourceServiceTest.java

```diff
@@ -433,14 +312,9 @@ void delete_shouldDeleteInstanceAndPublishResourceUpdatedEventWithNewAndOldWorks
 
     // then
     verify(resourceRepo).delete(instance);
-    var eventCaptor = ArgumentCaptor.forClass(ResourceUpdatedEvent.class);
-    verify(applicationEventPublisher).publishEvent(eventCaptor.capture());
-    assertThat(eventCaptor.getValue().newWork().getId()).isEqualTo(work.getId());
-    assertThat(eventCaptor.getValue().newWork().getIncomingEdges()).hasSize(1);
-    assertThat(eventCaptor.getValue().newWork().getIncomingEdges()).contains(edge2);
-    assertThat(eventCaptor.getValue().oldWork().getId()).isEqualTo(work.getId());
-    assertThat(eventCaptor.getValue().oldWork().getIncomingEdges()).hasSize(2);
-    assertThat(eventCaptor.getValue().oldWork().getIncomingEdges()).contains(edge1, edge2);
+    var resourceDeletedEventCaptor = ArgumentCaptor.forClass(ResourceDeletedEvent.class);
+    verify(applicationEventPublisher).publishEvent(resourceDeletedEventCaptor.capture());
+    assertEquals(instance, resourceDeletedEventCaptor.getValue().resource());
```

Comment:
suggest not to mix test libs and use assertThat api
change it to `assertThat(instance),isEqualTo(...`

Reply:
I agree that it's better to stick to one test lib only, but this was here before and it's just result of refactoring.
Fixed anyway, thx